lisa/tools/tar.py

```diff
@@ -6,13 +6,25 @@
 from assertpy import assert_that
 
 from lisa.executable import Tool
+from lisa.operating_system import Suse
+from lisa.tools.bzip2 import Bzip2
 
 
 class Tar(Tool):
     @property
     def command(self) -> str:
         return "tar"
 
+    def _check_exists(self) -> bool:
+        if (
+            isinstance(self.node.os, Suse)
+            and not self.node.tools[Bzip2]._check_exists()
```

Comment:
The Bzip2 can be added as a dependency of Tar, if it's required.

Reply:
Avoid to call private method of other classes. If it happens, it means the best practices are not followed, or the class design should be changed.

Reply:
This requirement is only for Suse. I checked on ubuntu and Redhat and it works fine

Reply:
When calling the `tools[Bzip2]`, the `_check_exists` is already called, so it's not useful to call `_check_exists`.